collect_app/src/test/java/org/odk/collect/android/utilities/CopyImageAndApplyExifRotationTest.kt

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2017 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.odk.collect.android.utilities
+
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import androidx.exifinterface.media.ExifInterface
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import timber.log.Timber
+import java.io.File
+import java.io.IOException
+
+@RunWith(AndroidJUnit4::class)
+class CopyImageAndApplyExifRotationTest {
```

Comment:
I think we should have these tests in `FileUtilsTest` or in a file prefixed with `FileUtils`. Otherwise, tools like IntelliJ's "switch to test" won't be able to find the test from the implementation and vice versa.

Reply:
I factored out `ImageUtils` class because `FileUtils` was big already, then I rename that class to `ImageUtilsTest` and merged with `BitmapScaledToDisplayTest` which also tested only methods from the same class.